### PR TITLE
Disable bandwidth-manager by default for new deployments

### PR DESCRIPTION
**Pull request by qmonnet**
Bandwidth manager is a beta feature in v1.9. Disable it by default for new deployments. Because it is already [disabled by default for the agent](https://github.com/cilium/cilium/blob/753231071fd29bca72b97e903f9337db4b17ee37/daemon/cmd/daemon_main.go#L536), this means we simply have to disable it on the Helm side.

The bandwidth manager is voluntarily re-enabled as a default for the `experimental-install.yaml` (through the Makefile), based on [feedback on Slack](https://cilium.slack.com/archives/C2B917YHE/p1602521560222000). File `quick-install.yaml` is updated.

Addresses one item for: https://github.com/cilium/cilium/issues/13371

### CODE REVIEW

**Comment by qmonnet** [comment]
Should I create an issue so we don't forget to re-enable it for 1.10?

**Review by pchaigno — APPROVED** [review]

**Comment by pchaigno** [comment]
test-me-please

**Comment by qmonnet** [comment]
test-gke

**Comment by kAworu** [comment]
let's wait on https://github.com/cilium/cilium/pull/13259 to be merged and rebase over please.

File: install/kubernetes/cilium/values.yaml
Comment (pchaigno) [inline install/kubernetes/cilium/values.yaml]: We'll also need to update the K8sBandwidthTest tests to explicitly enable the bandwidth manager:
https://github.com/cilium/cilium/blob/d6ad56b02764f32db482c56e8dc41c42671a93eb/test/k8sT/Bandwidth.go#L146-L167

**Review by pchaigno — CHANGES_REQUESTED** [review]

**Review by aanm — APPROVED** [review]
Changes LGTM except what Paul said. Also, we might need to wait until #13259 is merged

**Comment by borkmann** [comment]
Thanks @qmonnet . Please also update https://docs.cilium.io/en/latest/gettingstarted/bandwidth-manager/

**Comment by qmonnet** [comment]
Thank you all for the feedback and reviews. Now rebased on top of the Helm changes, with a separate commit to make sure the bandwidth manager is used for the related tests. 

I also opened a draft PR to revert the commit and re-enable the manager by default for v1.10: https://github.com/cilium/cilium/pull/13535

**Comment by qmonnet** [comment]
test-me-please

**Review by pchaigno — APPROVED** [review]

File: install/kubernetes/cilium/values.yaml
Comment (sayboras) [inline install/kubernetes/cilium/values.yaml:166]: If possible, maybe add docs here :thinking: , otherwise we can do it as part of #13496 as well :)

**Review by sayboras — APPROVED** [review]
LGTM :100:

File: install/kubernetes/cilium/values.yaml
Comment (qmonnet) [reply install/kubernetes/cilium/values.yaml:166]: The PR is about disabling the feature by default, and I intend to revert the related commit after the release is done. So I'd rather have the documentation added in another PR, if you don't mind.

**Comment by qmonnet** [comment]
test-gke

**Comment by qmonnet** [comment]
test-gke

**Review by tklauser — APPROVED** [review]

**Pull request merged**
